sc-tools/sc-config/src/sc_config.cpp

```diff
@@ -66,14 +67,48 @@ std::string ScConfigGroup::operator[](std::string const & key) const
   if (it == m_pathKeys.cend())
     return value;
 
-  if (value[0] == '/')
-    stream << value;
-  else
-    stream << m_configPath << value;
+  std::string valueStr(value);
+  std::string::size_type start = 0;
+
+  while (true)
+  {
+    std::string::size_type end = valueStr.find(ScConfig::PATHS_SEPARATOR, start);
+    std::string path = valueStr.substr(start, end - start);
+
+    path.erase(std::remove_if(path.begin(), path.end(), ::isspace), path.end());
+
+    if (!path.empty())
+    {
+      stream << NormalizePath(path);
+      stream << (end == std::string::npos ? "" : std::string(1, ScConfig::PATHS_SEPARATOR));
+    }
+
+    if (end == std::string::npos)
+      break;
+
+    start = end + 1;
+  }
 
   return stream.str();
 }
 
+std::string ScConfigGroup::NormalizePath(std::string const & path) const
```

Comment:
now something like this leads to errors appearing in logs
extensions = ./bin/extensions;/home/crafting-bench/work/main-sc-machine/bin/extensions